cohort-evaluator-spark/src/test/java/com/ibm/cohort/cql/terminology/R4FileSystemFhirTerminologyProviderTest.java

```diff
@@ -29,12 +29,17 @@
 import org.opencds.cqf.cql.engine.terminology.CodeSystemInfo;
 import org.opencds.cqf.cql.engine.terminology.ValueSetInfo;
 
+import com.ibm.icu.impl.Assert;
```

Comment:
This is a mistaken auto-import. We shouldn't be using any com.ibm.icu classes. You probably meant to grab `org.junit.Assert`

Reply:
Good catch, was wondering why the build kept barfing on that.